adapter/redisquota/driver.go

```diff
@@ -37,7 +39,9 @@ type response struct {
 
 // Get is to get a connection from the connection pool.
 func (cp *connPool) get() (*connection, error) {
+	cp.Lock()
```

Comment:
`Unknown quota 'RequestCount' requested` points to a missing quota descriptor in the global config - can you post your global config? [This is the key section.](https://github.com/istio/mixer/blob/master/testdata/globalconfig.yml#L78-L82)

`builder: name:"default" kind:"quotas" impl:"memQuota"` I think you're still getting memQuota and not redisQuota too. That should be at the top of the globalconfig in the [`adapters` section](https://github.com/istio/mixer/blob/master/testdata/globalconfig.yml#L3).